syntax = "proto3";
package MyRpc;

option cc_generic_services = true;

message RpcHeader {
    bytes service_name = 1;
    bytes method_name = 2;
    int32 args_size = 3;
}